packages/nocodb-sdk/src/lib/parser/common-cst-parser.ts

```
@@ -0,0 +1,92 @@
+    opt: ConstructorOpt = {
+      enabledRules: {
+        VARIABLE: 'VARIABLE',
+      },
+    }
+  ) {
+    super(tokens);
+    this.prepareCommonRules(opt);
+  }
+
+  protected prepareCommonRules(opt: ConstructorOpt) {
+    const $ = this;
```

Biome (1.9.4)

[error] This aliasing of this is unnecessary.

Arrow functions inherits `this` from their enclosing scope.
Safe fix: Use this instead of an alias.


(lint/complexity/noUselessThisAlias)